module/documents/actor.mjs

```diff
@@ -76,27 +81,50 @@ export default class ActorEd extends Actor {
 
   /**
    * Legend point History earned prompt trigger
+   * @returns {Promise<void>}
    */
-  async legendPointHistoryEarned( ) {
+  async legendPointHistoryEarned() {
     // let history = await getLegendPointHistoryData( actor );
-    const lpUpdateData = await LegendPointHistoryEarnedPrompt.waitPrompt( new LpTrackingData( this.system.lp.toObject() ), {actor: this} );
-    return this.update( {system: { lp: lpUpdateData }} )
+    const lpUpdateData = await LegendPointHistoryEarnedPrompt.waitPrompt( new LpTrackingData( this.system.lp.toObject() ), { actor: this } );
+    return this.update( { system: { lp: lpUpdateData } } )
   }
 
+  // eslint-disable-next-line max-params
+  async createEdRollOptions( testType = "action", rollType, strain = {}, target = {}, step = {}, devotionRequired, chatFlavor ) {
```

Comment:
Why is this async?

Also, this should probably be a `static` method of de `EdRollOptions` where you pass in this document...

Also, the signature should be shortened, where you're just passing the basic data in an object. Something like:
```javascript
/**
 @param options {edRollCreationOptionsblabla} object with options like `devotionRequired` or sth like that
 */
static createEdRollOptions( data, actor, options ) {
  // do stuff
}
```

Reply:
I don't really understand what this would mean

Reply:
![image](https://github.com/patrickmohrmann/earthdawn4eV2/assets/76265425/58c9f7ce-f270-4625-92d8-d2a506a8906b)